### PR TITLE
Update to Duktape 1.8.0.

### PR DESCRIPTION
**Pull request by szurbrigg**

### CODE REVIEW

**Comment by JakeWharton** [comment]
Holy horrid logs Batman. I'll look later in the week if no one beats me to it.

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ android:
     - platform-tools
     - tools
     - build-tools-25.0.0
-    - android-25
+    - android-18
```
Comment (JakeWharton) [inline .travis.yml]: What's this for? Doesn't look like you changed the compileSdkVersion?

Comment (szurbrigg) [inline .travis.yml]: Just stabbing away at the error I'm getting in Travis.  Not sure why it's a problem all of a sudden.

```android create avd --force -n test -t android-18 --abi armeabi-v7a
Valid ABIs: no ABIs.
Error: Invalid --abi armeabi-v7a for the selected target.```

**Comment by szurbrigg** [comment]
Thanks!

**Pull request merged**